src/assert/assert.ts

```diff
@@ -1,7 +1,17 @@
 import { AssertionError } from 'assert'
```

Comment:
Can we already just use the `assert` package directly or do we really need the wrapper Class in this file?

Reply:
Sure, nice point! Removed in favor of build in assert module.